be/src/olap/storage_engine.h

```diff
@@ -144,7 +144,7 @@ class StorageEngine {
     TxnManager* txn_manager() { return _txn_manager.get(); }
     MemTableFlushExecutor* memtable_flush_executor() { return _memtable_flush_executor.get(); }
 
-    bool check_rowset_id_in_unused_rowsets(const RowsetId& rowset_id);
+    bool check_rowset_id_in_unused_rowsets(const RowsetId& rowset_id, RowsetSharedPtr* rs);
```

Comment:
```suggestion
    bool check_rowset_id_in_unused_rowsets(const RowsetId& rowset_id, RowsetSharedPtr* rs = nullptr);
```
Add some comment of params

Reply:
done